### ISSUE TITLE
[Bug] t.rast.whatcsv uses broken r.what query

### ISSUE THREAD

**Issue by neteler** [OSGeo/grass-addons#490]
The usage and also the test of `t.rast.whatcsv` fails due to an incorrect `r.what` call:

```
GRASS :testsuite > python test_whatcsv.py 

ERROR: Raster map <dummy> not found
Traceback (most recent call last):
  File "/home/mneteler/.grass7/addons/scripts/t.rast.whatcsv", line 160, in <module>
    main(options, flags)
  File "/home/mneteler/.grass7/addons/scripts/t.rast.whatcsv", line 130, in main
    r_what = gcore.read_command("r.what", map="dummy",
  File "/home/mneteler/software/grass78/dist.x86_64-pc-linux-gnu/etc/python/grass/script/core.py", line 503, in read_command
    return handle_errors(returncode, stdout, args, kwargs)
  File "/home/mneteler/software/grass78/dist.x86_64-pc-linux-gnu/etc/python/grass/script/core.py", line 342, in handle_errors
    raise CalledModuleError(module=None, code=code,
grass.exceptions.CalledModuleError: Module run None r.what --q map=dummy output=- separator=| ended with error
Process ended with non-zero return code 1. See errors in the (error) output.
```

The reason is this call:

https://github.com/OSGeo/grass-addons/blob/4d03303a96cf3acf9f33f97c8bfe0cff03cb6cde/grass7/temporal/t.rast.whatcsv/t.rast.whatcsv.py#L130

which needs to be rewritten.

**Comment by neteler** [comment]
Bug still confirmed in GRASS 8.2.1dev

**Comment by petrasovaa** [comment]
This was fixed in #812, or is still something wrong with it?

**Comment by neteler** [comment]
You are right, I must have had an outdated addon version installed.

```
GRASS nc_spm_08_grass7/user1:testsuite > python test_whatcsv.py
1|115.0043586274|36.3593955783||100
2|79.6816763826|45.2391522853||200
3|97.4892579600|79.2347263950||300
4|115.0043586274|36.3593955783||400


.
----------------------------------------------------------------------
Ran 1 test in 5.480s
OK
```
